_src/css/_contact.scss

```diff
@@ -0,0 +1,40 @@
+.contact{
+
+    &__container{
```

Comment:
Although it works, you can redirect all the rules from contact__container to .contact.